react/AutocompleteBlock.tsx

```diff
@@ -115,9 +116,17 @@ const AutocompleteBlock: StorefrontFunctionComponent<any &
 
   const { selectedItem, quantitySelected, unitMultiplier } = state
   const callAddToCart = async (items: any) => {
+      let currentItemsInCart = orderForm.orderForm.items
     const mutationResult = await addToCart({
       variables: {
         items: items.map((item: any) => {
+            let existsInCurrentOrder = currentItemsInCart.filter(
```

Comment:
use `const` instead of `let` if you're not overwriting it
Use Array Destructuring             `const [existsInCurrentOrder] = currentItemsInCart.filter(`